aims/aim1.md

```diff
@@ -0,0 +1,44 @@
+# Aim 1:
+
+**Develop proof-of-concept deep learning methods to learn a latent space from single cell transcriptomic data.**
+
+## Background
+
+Deep unsupervised models can learn to approximate a data generating manifold often represented in a lower dimensional latent space.
+The latent space of such models, including Restricted Boltzmann Machines (RBMs), Variational Autoencoders (VAEs), and Generative Adversarial Networks (GANs), preserves non-linear distances between objects and learns to represent features driving variation in the input data [1,2,3].
+Thus, the algorithms serve a dual purpose: (1) non-linear dimensionality reduction and (2) the ability to generate hypothetical data.
```

Comment:
I think that `data generating manifold` may be too much jargon (first sentence). Someone who misses that is likely to be confused about 2 here. Do you just want to assert these two things in the first sentence and skip the `data generating manifold` terminology there?